SU-SWS/stanford_image_styles
# D8CORE-334: Responsive Image Styles

# READY FOR REVIEW

# Summary
- Updates breakpoint settings
- Updates responsive images
- Adds new responsive images for Card and CTA

# Review By (Date)
- Oct 18

# Urgency
- Low

# Steps to Test

1. Review Code
2. Check out this branch
3. Do a full Drupal site install
4. Run tests
5. Do a manual review of image styles

# Affected Projects or Products
- Hero (banner) paragraph
- More to come as I swap out the image style in the paragraph later

# Associated Issues and/or People
- D8CORE-334
-https://github.com/SU-SWS/acsf-cardinalsites/pull/29
- https://github.com/SU-SWS/stanford_profile/pull/25

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
